scripts/prepare_fsd50k.py

```diff
@@ -0,0 +1,159 @@
+import os
+import json
+import soundata
```

Comment:
make sure to include `soundata` in `setup.py` as an install.

Reply:
there's currently an issue with `soundata` and `numpy`. Let's keep track of this. Hopefully it will be resolved soon. https://github.com/soundata/soundata/issues/121